src/app/tenant/pages/tenant-search/tenant-search.component.ts

```diff
@@ -52,25 +52,6 @@ export class TenantSearchComponent implements OnInit {
           actionCallback: () => this.toggleChartVisibility(),
         },
       ];
-      if (vm.searchConfigEnabled) {
```

Comment:
Why have you removed this code? If your BFF is not implementing the search config endpoints they will do no harm but if someone implements them later, they are needed

Reply:
The search only includes one search parameter orgId. In my opinion its unnecessary to show the tab in the actions but I could add it again